v1.1/enable-node-map.md

```diff
@@ -0,0 +1,3 @@
+---
+redirect_to: ../v2.0/enable-node-map.md
```

Comment:
The redirect is failing because this needs to have the `.html` file ending, i.e., `redirect_to: ../v2.0/enable-node-map.html`.

Reply:
Ah! Thanks!